lib/views/home/home_view.dart

```diff
@@ -206,12 +207,13 @@ class _HomeViewState extends State<HomeView> {
                               child: Text(
                                 checkListIdentifier(
                                         listTile: snapshot.data![index])
-                                    ? snapshot.data![index].label
-                                    : snapshot.data![index].bolt11,
+                                    ? "Label of the invoice : ${snapshot.data![index].label}"
+                                    : "Amount Sent : ${snapshot.data![index].amountSent}",
```

Comment:
```suggestion
                                    ? "Label of the invoice : ${snapshot.data![index].label}"
                                    : "Amount Sent : ${snapshot.data![index].amountSent}",
```

You need to take a shot, or the amount or the text! I never see in my bank account a list with different title meanings, do you?

Or always the amount or always the text! This is up to you

Reply:
I think the amount would make more sense for a user, making it to label would make more sense for a bitcoin dev. What do you think?

Reply:
imho the label should be in the card body, not in the title because the card title should be always the same and in there you should put a value that you have always

Reply:
Yup, I think the same